### PR TITLE
Import Priority Area Polygons

### LINKED ISSUE #1275: Import GeoJSON polygons for Priority Areas
I would like to be able to use a GeoJSON polygon file to show the Priority Areas in addition to drawing a polygon/rectangle/circle.
image

### PR DESCRIPTION
**Pull request by d-rita**
Fixes #1275

### CODE REVIEW

File: frontend/src/components/projectCreate/index.js [hunk 1]
```diff
@@ -30,10 +30,80 @@ const ProjectCreationMap = React.lazy(() =>
   import('./projectCreationMap' /* webpackChunkName: "projectCreationMap" */),
 );
 
-var toGeojson = require('@mapbox/togeojson');
+var toGeojson = require('@tmcw/togeojson');
```
Comment (willemarcel) [inline frontend/src/components/projectCreate/index.js:33]: With the new version, we can do the `import toGeojson from "@tmcw/togeojson";`

File: frontend/src/components/projectCreate/index.js [hunk 2]
```diff
@@ -30,10 +30,80 @@ const ProjectCreationMap = React.lazy(() =>
...
 var osmToGeojson = require('osmtogeojson');
 var shpjs = require('shpjs');
 
+export const readGeoFile = (e, format, error) => {
```
Comment (willemarcel) [inline frontend/src/components/projectCreate/index.js:37]: Let's move `readGeoFile` to the utils folder?

**Comment by willemarcel** [comment]
Let's merge first #4407, then rebase this PR and make the adjustments.

**Comment by d-rita** [comment]
> Let's merge first #4407, then rebase this PR and make the adjustments.

Alright, will do

**Review by d-rita — COMMENTED** [review]
The changes look good, @willemarcel

File: frontend/src/components/projectCreate/index.js (hunk 1 above)
Comment (d-rita) [reply frontend/src/components/projectCreate/index.js]: Worked on it. I used `import { kml } from "@tmcw/togeojson";`

**Review by willemarcel — APPROVED** [review]
Awesome work!

**Pull request merged**